core/rpc/README.md

```diff
@@ -1700,6 +1701,170 @@ echo '{
 }
 ```
 
+### Method `build_sudt_issue_transaction`
+
+- `build_sudt_issue_transaction(owner, source, to, pay_fee, change, fee_rate, since)`
+  - `owner`: `string`
+  - `source`: `"free"|"claimable"`
```

Comment:
The `source` is unnecessary. Because `build_sudt_issue_transaction` is to create sUDT, do not need to consume sUDT cell such as cheque cell.

Reply:
You're right. The `source` parameter has been removed.